res/css/structures/_RightPanel.pcss

```diff
@@ -25,6 +25,7 @@ limitations under the License.
     box-sizing: border-box;
     height: 100%;
     contain: strict;
+    background-color: var(--cpd-color-bg-canvas-default);
```

Comment:
This stops extra broken stacking of elements when right panel starts overlapping the room header at very small widths.